ap_api/resources/index.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Resources
+sidebar_position: 20
+---
+
+import {MethodTable} from "@site/src/components/MethodTable";
+
+Data on the Anchor Platform is organized according to resources. Each resource has several different endpoints.
+
+<MethodTable title="Resource Types">
+
+| | |
+| --------------------------------------------------- | --- |
+| [Transactions](./transactions/index.mdx) | |
+| [Quotes](./quotes/index.mdx) | |
```

Comment:
Maybe switch the order of Transactions and Quotes to match with the order in the sidebar?